data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			`	The captain scoffs. "Well yeah, but I figured it was just a coverup from the Navy to stop people from coming here."`
+			`	"Would that not imply that the secret isn't truly 'out' then?" Sayari asks while giving the captain a stern look.`
+				goto trouble
+			label xilin
+			`	"No, I don't know," says Xilin while giving the captain a stern look. "Rumors of the Hai becoming mainstream don't constitute the secret being 'out' to me."`
+			label trouble
+			`	"Well when you put it that way," the captain says nervously. "Ok, just spit it out already. Am I in trouble?"`
+			`	"No," Xilin says while looking away. Something else has caught his attention. "But they might be."`
+			`	You look up to see a beat-up Berserker that looks like it can barely fly landing in the spaceport. "You're free to go," Xilin says to the Star Queen captain.`
+			`	You approach the Berserker as it finishes landing. The captain of the ship exits looking far calmer than the condition of the Berserker would suggest its captain should be. "Uh, hi," he says akwardly. "Can I help you guys?"`
+			choice
+				`	"What happened to your ship?"`
+				`	"Did you get attacked by pirates on the way here."`
+			`	The captain looks back at the poor state of his interceptor. "Oh, that. Yeah, I ran into a couple pirates on the way here. Nothing I haven't dealt with before, though."`
+			`	"So you've been to Hai space before?" Xilin asks.`
+			`	"Sure. Plenty of times," he responds.`
+			`	Xilin looks over the interceptor with a quizzical expression. "May I see your pilot's license?"`
+			`	"Yeah, no problem," the captain says, as he quickly pulls a license out of his pocket that looks almost as beat up as the interceptor. Xilin jots down the details and hands the license back to the captain. "Thank you. And maybe tag along with a merchant fleet on your way back home."`
+			`	"That sounds like a good idea," the captain says before walking off into the spaceport.`
+			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Some already knew of the Hai before, but many are new arrivals who heard of the way to the Hai through whispers in spaceport bars, or were sold coordinates by knowledgeable captains. Everyone seems peaceful enough in interacting with the Hai, but clearly the word is getting out more as time goes on.`
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		conversation
+			`You return to the briefing room at the secure port with Xilin and Sayari, and Xilin begins frantically copying notes into a terminal. He then pauses while staring at something on the screen.`
+			`	"I'm running each of the pilot's licenses and ship registrations through my copy of the Republic's pilot database. They've all come up as valid and with mostly clean records. All except for one." He turns the terminal screen toward you and Sayari, displaying an inputted license ID that the system says does not exist.`
```

Comment:
In theory he could have done most of this work on a portable device (since he'd have to have a copy of the database not stored on the Hai's system anyway) so this frantic action doesn't make a lot of sense.

Reply:
> I'd presume this terminal is the location of that database, instead of walking around with such sensitive information

This is what I was thinking. I also imagine this terminal as also being air-gapped with no wireless connections, so it isn't something he could sync with. Note too that at this point in the story, all communications across the wormhole need to be done manually, as there's no sort of connector running through the wormhole to allow information to travel wirelessly, so any digital information that Xilin wants to access he had to physically bring with him.

Outside of the story, though, I just needed some way of having this conversation take place during the on complete, instead of loading everything into the stopover conversation and having nothing left to talk about once we got here.

I'll change the first line to say that he transfers the information into the terminal instead of frantically copying it.